developer/js/libexec/create-override-script-regexp.ts

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env npx ts-node
+import {readFileSync} from "fs";
+import * as path from "path";
+
+// Where to find UnicodeData.txt and Blocks.txt
+const UCD_DIR = path.join("..", "..", "resources", "standards-data", "unicode-character-database");
+
+const SPACELESS_SCRIPT_BLOCKS = new Set([
+  "Myanmar",
+  "Lao",
+  "Thai",
+  "Khmer",
+  "Katakana",
+  "Katakana Phonetic Extensions",
+  // Add more scripts here, as necessary!
+]);
+
+let blockIter = blocks();
+let block = nextBlock()
+
+let elligibleCharacters = [];
```

Comment:
```suggestion
let eligibleCharacters = [];
```
and elsewhere of course 😁